addons/l10n_it_edi/models/ir_mail_server.py

```diff
@@ -47,7 +60,16 @@ def fetch_mail(self):
                 imap_server = server.connect()
                 imap_server.select()
 
-                result, data = imap_server.uid('search', None, '(FROM "@pec.fatturapa.it")', '(UID %s:*)' % (server.l10n_it_last_uid))
+                # Only download new emails
+                email_filter = ['(UID %s:*)' % (server.l10n_it_last_uid)]
+
+                # The l10n_it_edi.fatturapa_bypass_incoming_address_filter prevents the sender address check on incoming email.
+                bypass_incoming_address_filter = self.env['ir.config_parameter'].get_param('l10n_it_edi.bypass_incoming_address_filter', False)
+                if not bypass_incoming_address_filter:
+                    email_filter.append('(FROM "@pec.fatturapa.it")')
+
+                dummy, data = imap_server.uid('search', None, *email_filter)
```

Comment:
```suggestion
                data = imap_server.uid('search', None, *email_filter)[1]
```

You might have a pylint warning for unused variable without it, not sure

Reply:
"Dummy" variable prevents pylint to give a warning, and I tried to be "minimal" with changes.

Reply:
I can understand the dummy if you don't care about a variable in the middle or something, but here I don't see why it would be less "minimal" to explicitly take only the second element of the return value.

```python
useful1, dummy, useful2 = my_semi_useful_method()
```

If the CI/style doesn't complain, then whatever both are fine

Reply:
I've fixed as you suggested, but left other occurrences of the same "unused variable" problems unfixed, to have a less cluttered diff.